src/show-debugger.js

```diff
@@ -64,4 +66,18 @@ export default function(context) {
       },
     },
   })
+
+  // Keeping the panel at the default window level
+  webUI.panel.setLevel(NSNormalWindowLevel)
+  // I found that keeping it at a window level allows me to
+  // interact with Sketch freely without having the debugger
+  // always in the way. I can switch to the debugger when needed
+  // with the native 'Move focus to next window' shortcut.
+  // IDEA: Add level/position/size options to debugger window?
+  // IDEA: Add option to 'dock to bottom' à la chrome devtools?
+  // IDEA: Keeping the window always on top is handy for debuggin the debugger
```

Comment:
regarding "Keeping the window always on top is handy for debuggin the debugger", a way to do it would be
```js
if (process.env.NODE_ENV === 'production') {// that's when we ship the plugin
  webUI.panel.setLevel(NSNormalWindowLevel
}
```